app/js/immedia/immedia-room-controller.js

```diff
@@ -6,7 +6,18 @@
 
   var immediaControllers = angular.module('immediaControllers');
 
-  immediaControllers.controller('RoomCtrl', ['$rootScope', '$scope', '$sce', '$window', '$routeParams', '$interval', 'RoomService', 'ConfigService', function($rootScope, $scope, $sce, $window, $routeParams, $interval, roomSvc, cfgSvc) {
+  immediaControllers.controller('RoomCtrl', ['$rootScope',
```

Comment:
I'd prefer having this list in a single line, with a break before the function definition maybe.